example/speech-demo/lstm_bucketing.py

```diff
@@ -0,0 +1,324 @@
+# pylint:skip-file
+import sys
+sys.path.insert(0, "../../python")
+import mxnet as mx
+import numpy as np
+import mxnet.ndarray as ndarray
+
+from lstm import LSTMState, LSTMParam, LSTMModel, lstm
+
+import logging
+import validictory
+import math
+import numpy
+import time
+
+from io_func.feat_io import DataReadStream
+from utils.utils import setup_logger, to_bool
+from utils.main_runner import run_main
+DATASETS = {}
+
+DATASETS["AMI_train"] = {
+        "lst_file": "/data/sls/scratch/yzhang87/AMI/ami_sdm_baseline/exp_cntk/sdm1/cntk_train_mxnet.feats",
```

Comment:
Can you add to the README file that

- those paths need to be changed before running the demo
- where can one find and download the dataset (if publicly available)